pkg/scheduler/schedule_one_test.go

```diff
@@ -20,6 +20,7 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"github.com/stretchr/testify/assert"
```

Comment:
we avoid this library in sig-scheduling

Reply:
ok, I will remove this.

BTW, May I ask if there are any issues or risks with this library? Since I used this library in my personal project.

Reply:
Personal preference, for consistency with the existing code.

`t.Error` and `t.Fatal` allow to express clearly what went wrong and whether we should proceed with the rest of the test.

Reply:
ok, I got it. Thanks.